impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -31,28 +32,27 @@
 import com.okta.sdk.client.Proxy;
 import com.okta.sdk.impl.api.ClientCredentialsResolver;
 import com.okta.sdk.impl.api.DefaultClientCredentialsResolver;
-import com.okta.sdk.authc.credentials.ClientCredentials;
-import com.okta.sdk.impl.config.ClientConfiguration;
-import com.okta.sdk.impl.config.EnvironmentVariablesPropertiesSource;
-import com.okta.sdk.impl.config.OptionalPropertiesSource;
-import com.okta.sdk.impl.config.PropertiesSource;
-import com.okta.sdk.impl.config.ResourcePropertiesSource;
-import com.okta.sdk.impl.config.SystemPropertiesSource;
-import com.okta.sdk.impl.config.YAMLPropertiesSource;
+import com.okta.sdk.impl.api.OAuth2ClientCredentialsResolver;
+import com.okta.sdk.impl.config.*;
 import com.okta.sdk.impl.http.authc.RequestAuthenticatorFactory;
 import com.okta.sdk.impl.io.ClasspathResource;
 import com.okta.sdk.impl.io.DefaultResourceFactory;
 import com.okta.sdk.impl.io.Resource;
 import com.okta.sdk.impl.io.ResourceFactory;
+import com.okta.sdk.impl.oauth2.AccessTokenRetrieverService;
+import com.okta.sdk.impl.oauth2.AccessTokenRetrieverServiceImpl;
+import com.okta.sdk.impl.oauth2.OAuth2AccessToken;
+import com.okta.sdk.impl.oauth2.OAuth2ClientCredentials;
 import com.okta.sdk.impl.util.DefaultBaseUrlResolver;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.LinkedHashMap;
-import java.util.Map;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.util.*;
```

Comment:
nit: remove the * import.

Personally I don't mind them in tests, but we should probably list them out here

Reply:
IntelliJ groups it and puts a * :)

Reply:
there is an option in your editor to fix that ;)

Reply:
Yes, I figured :)